### PR TITLE
Add support multipart/form-data

### PR DESCRIPTION
**Pull request by maranqz**
Resolving issue #118

### CODE REVIEW

File: apitest.go [hunk 1]
```diff
@@ -221,6 +225,8 @@ type Request struct {
 	queryCollection map[string][]string
 	headers         map[string][]string
 	formData        map[string][]string
+	multipartBody   *bytes.Buffer
+	multipart       *multipart.Writer
```
Comment (maranqz) [inline apitest.go:229]: Should multipartBody/multipart be replaced on?
```
multipartFile         map[string][]string
multipartValue      map[string][]string
```

**Comment by steinfletcher** [comment]
Thanks for the contribution. I'm currently away but will find time to get to this soon!

**Comment by maranqz** [comment]
Hello, I just remaind about the PR:)

**Comment by maranqz** [comment]
Hello, I'm just reminding about PR :)

**Comment by steinfletcher** [comment]
> Hello, I'm just reminding about PR :)

Sorry for the delay. 

> Hello, I'm just reminding about PR :)

Sorry for the delay. I've merged it. I'll cut a release Sunday evening. Many thanks

**Pull request merged**